src/plone/restapi/deserializer/site.py

```diff
@@ -10,68 +10,85 @@
 
 import json
 
+try:
+    from Products.CMFPlone.factory import PLONE60MARKER
+
+    PLONE60MARKER  # pyflakes
+except ImportError:
+    PLONE_6 = False
+else:
+    PLONE_6 = True
```

Comment:
This is code already in `plone.restapi`:
```suggestion
from pkg_resources import get_distribution
from pkg_resources import parse_version


HAS_PLONE_6 = parse_version(
    get_distribution("Products.CMFPlone").version
) >= parse_version("6.0.0a1")
```

See https://github.com/plone/plone.restapi/search?q=HAS_PLONE_6

Reply:
@ale-rt I know, that was before we had PLONE60MARKER in place (which I took care of).

Maybe we should change the old one?

Reply:
For me, using the marker is good. I approved adding it. :-).
It is similar to the `have plone-6` condition that we have in zcml.

Reply:
:) It seems you and @jensens wrote it. I pushed for a programmatic PLONE60MARKER because we were already using it in ZCML and we were using this way with `_IMREALLYPLONE5` in a number of tests already. I thought that it would be great to have, instead of dealing with parsing versions. Both are ugly enough and require a copy/paste (not easy recallable) tbh...

```
try:
    from Products.CMFPlone.factory import PLONE60MARKER  # noqa
except ImportError:
    PLONE_6 = False
else:
    PLONE_6 = True
```

Reply:
For me using that marker like this is a bad practice, if you really want to use it I suggest to use something like:
```
from importlib import import_module

HAS_PLONE_6 = getattr(import_module("Products.CMFPlone.factory"), "PLONE60MARKER", False)
```
which does not pollute the locals...

Reply:
@ale-rt Looks good to me! Far less ugly :) Let me try!